packages/react-scripts/config/paths.js

```diff
@@ -46,6 +46,8 @@ function getServedPath(appPackageJson) {
   return ensureSlash(servedUrl, true);
 }
 
+var doComponentModulesRegex = /node_modules[\\/]@digital-origin[\\/]do-component-/;
```

Comment:
Did you test that this url matches what you want?

If you tested it perfect!

If not maybe you'll need to change the regex to:

/node_modules[\\/]@digital-origin[\\/]do-component-.*/

Reply:
yes, i test it i dont know why you have to escape \, but you have to escape it